<launch>
  <arg
    name="model" />
  <param
    name="robot_description"
    textfile="$(find total_assembly_gripper)/urdf/total_assembly_gripper.urdf" />
  <!-- <node
    name="joint_state_publisher_gui"
    pkg="joint_state_publisher_gui"
    type="joint_state_publisher_gui" /> -->
    <node name="initial_joint_state_publisher" pkg="total_assembly_gripper" type="initial_config.py" output="screen"/>
    <!-- <node name="joint_state_publisher" pkg="total_assembly_gripper" type="set_desired_position.py" output="screen"/> -->
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="robot_state_publisher" />
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find total_assembly_gripper)/rviz/urdf.rviz" />
</launch>
